contracts/orderbook/IWalletFactor.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.8.4;
+
+interface IWalletFactory {
+    enum OrderType {
+        LIMIT_OPEN,
+        LIMIT_CLOSE,
+        MARKET_STOP
+    }
+    struct OpenOrder {
```

Comment:
I would call struct Order, and orderStatus is one of its fields, this way in the future you might have orderStauts enum (Filled, Open, PartFilled etc)